contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -161,14 +185,13 @@ contract ValidatorShare is IValidatorShare, ERC20NonTransferable, OwnableLockabl
         uint256 rate = exchangeRate();
         uint256 shares = _amount.mul(EXCHANGE_RATE_PRECISION).div(rate);
         require(shares >= _minSharesToMint, "Too much slippage");
-        require(delegators[msg.sender].shares == 0, "Ongoing exit");
+        require(unbonds[msg.sender].shares == 0, "Ongoing exit");
 
         _mint(msg.sender, shares);
 
         _amount = _amount.sub(_amount % rate.mul(shares).div(EXCHANGE_RATE_PRECISION));
 
-        totalStake = totalStake.add(_amount);
-        amountStaked[msg.sender] = amountStaked[msg.sender].add(_amount);
+        // totalStake = totalStake.add(_amount);
```

Comment:
Why `totalStake` removed? How we are going to get `totalStake` for this validator?